server/src/main/java/org/elasticsearch/index/mapper/ObjectMapper.java

```diff
@@ -362,17 +368,8 @@ public void toXContent(XContentBuilder builder, Params params, ToXContent custom
             custom.toXContent(builder, params);
         }
 
-        // sort the mappers so we get consistent serialization format
```

Comment:
Why is it okay to remove this? Shouldn't the serialization format still be consistent?

Reply:
this affected the result of `testUpdateNewColumnTableDefault`.
`["id", "score", "name", "good"]` is expected but due to sorting, it becomes `["id", "score", "good", "name"]`.
Could you help confirm the testcase is valid? My guess is that it is not and this change can be reverted.